mappings/net/minecraft/block/TargetBlock.mapping

```diff
@@ -1,5 +1,7 @@
 CLASS net/minecraft/class_4850 net/minecraft/block/TargetBlock
 	FIELD field_22428 POWER Lnet/minecraft/class_2758;
+	FIELD field_31263 PROYECTILE_POWER_DELAY I
```

Comment:
```suggestion
	FIELD field_31263 PROJECTILE_POWER_DELAY I
```

Reply:
I always write it with "y" as in Spanish lol